`ravel` function not working

To convert to a 1-d array I use ravel (see  https://docs.scipy.org/doc/numpy/reference/generated/numpy.ravel.html)

But this does not work:

```cpp
#define XTENSOR_ENABLE_ASSERT 1

#include <xtensor/xtensor.hpp>
#include <xtensor/xarray.hpp>
#include <xtensor/xrandom.hpp>
#include <xtensor/xmath.hpp>
#include <xtensor/xio.hpp>
#include <xtensor/xmanipulation.hpp>

int main()
{
  xt::xarray<int> a = {{0,1,0},{0,1,0}};
  std::cout << xt::ravel(a) << std::endl;
}
```

```none
test.cpp:13:16: error: no matching function for call to 'ravel'
  std::cout << xt::ravel(a) << std::endl;
               ^~~~~~~~~
/usr/local/include/xtensor/xmanipulation.hpp:285:17: note: candidate template ignored: couldn't infer template argument 'L'
    inline auto ravel(E&& e)
                ^
1 error generated.
```

--- a/include/xtensor/xmanipulation.hpp
+++ b/include/xtensor/xmanipulation.hpp
@@ -31,7 +31,7 @@ namespace xt
     template <class E, class S, class Tag = check_policy::none>
     auto transpose(E&& e, S&& permutation, Tag check_policy = Tag());
 
-    template <layout_type L, class E>
+    template <layout_type L = layout_type::row_major, class E>
     auto ravel(E&& e);
 
     template <class E>
@@ -278,7 +278,8 @@ namespace xt
     /**
      * Returns a flatten view of the given expression. No copy is made.
      * @param e the input expression
-     * @tparam L the layout used to read the elements of e
+     * @tparam L the layout used to read the elements of e. If no parameter
+     * is specified, layout_type::row_major is used.
      * @tparam E the type of the expression
      */
     template <layout_type L, class E>

--- a/test/test_xmanipulation.cpp
+++ b/test/test_xmanipulation.cpp
@@ -99,6 +99,9 @@ namespace xt
 
         auto flat2 = flatten(a);
         EXPECT_EQ(flat, flat2);
+
+        auto flat3 = ravel(a);
+        EXPECT_EQ(flat, flat3);
     }
 
     TEST(xstrided_view, split)

